hail/hail-ci-build.sh

```diff
@@ -194,6 +194,22 @@ test_gcp() {
     touch ${GCP_SUCCESS}
 }
 
-test_project
-
-test_gcp > ${GCP_LOG}
+test_project &
+PID1=$!
+echo "test_project PID is ${PID1}"
+
+test_gcp > ${GCP_LOG} &
+PID2=$!
+
+echo "test_gcp PID is ${PID2}"
+
+# collect parallel processes and fail if any fail
+for pid in "${PID1}" "${PID2}"; do
+    echo "waiting on PID ${pid}..."
+    wait $pid
```

Comment:
GCP will be reported as failed if it test_project fails first and we abort.  Probably not what we want.